build.gradle

```diff
@@ -1,20 +1,20 @@
 plugins {
   id 'application'
   id 'checkstyle'
-  id 'io.spring.dependency-management' version '1.0.3.RELEASE'
+  id 'io.spring.dependency-management' version '1.0.5.RELEASE'
   id 'org.flywaydb.flyway' version '5.0.7'
-  id 'org.springframework.boot' version '1.5.7.RELEASE'
+  id 'org.springframework.boot' version '1.5.12.RELEASE'
```

Comment:
spring boot 2 upgrade to come?

Reply:
Yes. That's a different ball game left aside